### PR TITLE
Get Trashed Items

### PR DESCRIPTION
**Pull request by marek-vysoky**
Adding comments, adding unit test

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @marek-vysoky has signed the CLA. Thanks for the pull request!

**Comment by coolboy** [comment]
@marek-vysoky Seems this pull has conflict, could you fix this?

**Comment by marek-vysoky** [comment]
I've rebased master to this branch, resolved the conflict and push the changes

File: Box.V2/Managers/BoxFoldersManager.cs [hunk 1]
```diff
@@ -250,7 +253,11 @@ public BoxFoldersManager(IBoxConfig config, IBoxService service, IBoxConverter c
         /// attributes can be passed in separated by commas e.g. fields=name,created_at. Paginated results can be 
         /// retrieved using the limit and offset parameters.
         /// </summary>
-        /// <returns></returns>
+        /// <param name="id">This param is not used in implementation</param>
+        /// <param name="limit">The maximum number of items to return</param>
+        /// <param name="offset">The maximum number of items to return</param>
```
Comment (coolboy) [inline Box.V2/Managers/BoxFoldersManager.cs]: wrong comment?

Comment (marek-vysoky) [inline Box.V2/Managers/BoxFoldersManager.cs]: This method is Obsolete and input param "id" is not used and is not sent as param and does not matter what value includes.

**Comment by marek-vysoky** [comment]
I've changed comment for offset

**Pull request merged**